test/any-api/apiconsumer.ts

```diff
@@ -0,0 +1,74 @@
+import { ethers } from 'hardhat';
```

Comment:
Should the filename be apiconsumer.ts or api-consumer.ts? I think kebab case is preferred in TS.